include/mbedtls/pk_info.h

```diff
@@ -237,6 +275,36 @@ struct mbedtls_pk_info_t
 
 };
 
+/**
+ * Methods that opaque key pair objects must implement.
+ *
+ * \brief Initializer for opaque key engines
+ *
+ * The value of this macro is a suitable initializer for an object of type
+ * mbedtls_pk_info_t. It is guaranteed to remain so in future versions of the
+ * library, even if the type mbedtls_pk_info_t changes.
+ *
+ * This macro is suitable for static initializers provided that all of its
+ * parameters are constant.
+ *
+ * \param name For transparent keys, this reflects the key type. For opaque
+ * keys, this reflects the cryptographic module driver.
+ * \param get_bitlen \ref mbedtls_pk_info_t.get_bitlen method
```

Comment:
`\ref mbedtls_pk_info_t.get_bitlen` is typeset as `mbedtls_pk_info_t::get_bitlen`. This works for C++, but not for C. If there's no simple way to typeset `get_bitlen` with a link to the `get_bitlen` field, just use `\c get_bitlen` and let people follow the link to the `mbedtls_pk_info_t` type. Please use `#mbedtls_pk_info_t` in the general description so that there's a link to the type definition.

Reply:
I'm not sure if I follow. That's how it looks after generating the apidoc:
![image](https://user-images.githubusercontent.com/29195988/36245245-1b484034-1222-11e8-9795-b0644eda43b3.png)
It's a link to the according field in the structure. Wasn't that the point?

Reply:
The point is that it's typeset as `mbedtls_pk_info::field_name`, but you can't write `mbedtls_pk_info::field_name` in C. The target of the link is fine, but the rendered text isn't. That's not a big deal though.